Sources/DistributedActors/Props.swift

```diff
@@ -74,7 +74,7 @@ public struct _Props: @unchecked Sendable {
 
     /// Allows for passing properties to creating a distributed actor.
     @TaskLocal
-    internal static var forSpawn: _Props = .init()
+    public static var forSpawn: _Props = .init()
```

Comment:
I don't think we can make it public, especially not without making `_underscored`.

We'll need something like it though... I'll think a bit how we want to expose it